apps/spruce/src/pages/version/Tabs/TimeChart/VariantTimeChart/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { useQuery } from "@apollo/client";
+import { useLocation } from "react-router-dom";
+import { useToastContext } from "@evg-ui/lib/context/toast";
+import { DEFAULT_POLL_INTERVAL } from "constants/index";
+import {
+  VersionGanttChartQuery,
+  VersionGanttChartQueryVariables,
+} from "gql/generated/types";
+import { VERSION_GANTT_CHART } from "gql/queries";
+import { usePolling } from "hooks";
+import { useQueryVariables } from "../../useQueryVariables";
+import TimeChartGraph from "../TimeChartGraph";
+import { transformVersionDataToVariantGanttChartData } from "../utils";
+
+interface Props {
+  versionId: string;
+}
+
+const VariantTimeChart: React.FC<Props> = ({ versionId }) => {
```

Comment:
I don't its necessary to have a second VariantTimeChart component. This is just a subset of whats shown in TaskTimeChart. We can just bake the additional variant filtering logic into the original component and reuse it.

Reply:
I think the same query can be used for both. `VersionTaskDurations` takes in variant as a parameter so you can technically just reuse the same query and add an optional variant filter. Its how the task-duration tab works.